src/Check/SecurityAdvisory.php

```diff
@@ -22,9 +22,9 @@ class SecurityAdvisory extends AbstractCheck
     protected $lockFilePath;
 
     /**
-     * @var \Enlightn\SecurityChecker\AdvisoryAnalyzer
+     * @var \Enlightn\SecurityChecker\AdvisoryAnalyzer|null
      */
-    protected $advisoryAnalyzer;
+    protected $advisoryAnalyzer = null;
```

Comment:
Do we need this state at all now? In theory, it can be a local variable, right?

Reply:
Yeah, I thought about it - but in this way the class can still be extended by this class with a setter: https://github.com/laminas/laminas-diagnostics/blob/1.11.x/test/TestAsset/Check/SecurityAdvisory.php to keep the tests here https://github.com/laminas/laminas-diagnostics/blob/1.11.x/test/ChecksTest.php#L558 valid.